### PR TITLE
[bitnami/mysql] Update to mysql chart to have startdb script

### LINKED ISSUE #17718: [bitnami/mysql] initdbScriptsConfigMap find: '/docker-entrypoint-startdb.d/': No such file or directory
### Name and Version

bitnami/mysql 9.7.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

kubernetes.
1 create a configmap from a mysql dump
```
./kubectl create configmap mysql-scheme --from-file=initdb.sql  || echo "configmap already exists"
```
2 use the helm options to install the mysql chart
```
helm upgrade -i mysql bitnami/mysql \
            --set auth.rootPassword=$MYSQL_DB_ROOT_PASS \
            --set auth.username=$MYSQL_DB_USER \
            --set auth.password=$MYSQL_DB_USER_PASS \
            --set auth.database=$MYSQL_DB_NAME \
            --set initdbScriptsConfigMap="mysql-scheme" \
            --version 9.7.1
```            

### Are you using any custom parameters or values?

yes written above ^^

### What is the expected behavior?

the mysql initialization should apply the config map sql script to the database creating tables

### What do you see instead?
```
mysql 21:29:17.05 INFO  ==> Starting mysql in background
find: '/docker-entrypoint-startdb.d/': No such file or directory
mysql 21:29:33.16 INFO  ==> Stopping mysql
```

### Additional information

I'm not sure if this is an issue or i'm doing something incorrectly...
so if there is a better place for me to ask questions to the community, please let me know, 
and thank u!

### PR DESCRIPTION
**Pull request by SyedMHashim**
### Description of the change
Added capability of specifying startdb scripts using helm values.
### Benefits
This change allows users to specify startdb scripts which would run every time the container starts. A possible usecase could be that the users can run specific checks on every container restarts and perform some actions accordingly.
### Possible drawbacks
Users needs to be carefully specify these scripts since they would run on every container restart.
### Applicable issues

- fixes #17718

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/mysql/Chart.yaml [hunk 1]
```diff
@@ -27,4 +27,4 @@ maintainers:
 name: mysql
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/mysql
-version: 9.10.10
+version: 9.10.11
```
Comment (migruiz4) [inline bitnami/mysql/Chart.yaml:30]: As this PR adds new features, could you please bump a minor version (9.11.0)?

File: bitnami/mysql/templates/primary/startdb-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+*/}}
+
+{{- if and .Values.startdbScripts (not .Values.startdbScriptsConfigMap) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-start-scripts" (include "mysql.primary.fullname" .) }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: primary
```
Comment (migruiz4) [inline bitnami/mysql/templates/primary/startdb-configmap.yaml:13]: I think this label does not fit here, could you please remove it?

File: bitnami/mysql/Chart.yaml (hunk 1 above)
Comment (SyedMHashim) [reply bitnami/mysql/Chart.yaml]: Done

File: bitnami/mysql/templates/primary/startdb-configmap.yaml (hunk 2 above)
Comment (SyedMHashim) [reply bitnami/mysql/templates/primary/startdb-configmap.yaml]: Removed

**Comment by SyedMHashim** [comment]
Hi @migruiz4. I had updated the code as per your request. Please let me know if there's anything else that needs to be changed.

**Review by migruiz4 — APPROVED** [review]
Thank you for applying the suggested changes, looks good to me!

**Pull request merged**